### DISCUSSION TITLE
Latest version?

### DISCUSSION THREAD

**Discussion by kickerofelves** [emacsorphanage/keychain-environment#8] [Support]
I recently installed keychain-environment. Now whenever I go into list-packages emacs wants to delete the 2.4.1 released in January 2019 and install a version apparently from March 2018. What's going on? What's the most recent version?

```
I keychain-environment           20180318.2223  available    melpa    load keychain environment variables
D keychain-environment           2.4.1          installed             load keychain environment variables
```

**Comment by tarsius** [comment]
- 20180318 is when I created the commit and tag using Git/Magit.
- January 2019 is when I clicked on "create release" in the GitHub web ui. That date has no significance.
